opus_codec_registration.src.html

```diff
@@ -132,13 +132,21 @@
 dictionary OpusEncoderConfig {
   OpusBitstreamFormat format = "opus";
   [EnforceRange] unsigned long frameDuration;
+  [EnforceRange] unsigned long complexity;
+  [EnforceRange] unsigned long packetlossperc;
```

Comment:
We should figure out if we want to be consistent in naming here. `frameDuration` is camel case, it seems like these should be `packetLossPerc`, `useInbandFec`, and `useDtx` -- if we're intentionally not doing that for consistency with webrtc or elsewhere it'd be good to note that in the commit at least.

Reply:
I went with no caps to be consistent with [section 6.1 of RFC758](https://datatracker.ietf.org/doc/html/rfc7587#section-6.1:~:text=useinbandfec%3A%20%20specifies%20that,default%20is%200.), and the [ORTC spec](https://draft.ortc.org/#opus-codec-settings*).

Reply:
We modeled the Opus parameter names in ORTC after RFC 7587 Section 6.1, since that was also used in the WebRTC API SDP.

Reply:
Do you want to define defaults for these as well?

Reply:
I will leave no-caps for now, but will update if #588 resolves one way or the other.